Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -164,6 +171,19 @@ void ABenchmarkGymGameMode::ParsePassedValues()
 	UE_LOG(LogBenchmarkGymGameMode, Log, TEXT("Density %d, Clusters %d"), PlayerDensity, NumPlayerClusters);
 }
 
+void ABenchmarkGymGameMode::BuildExpectedActorCounts()
+{
+	Super::BuildExpectedActorCounts();
+
+	AddExpectedActorCount(NPCBPClass, TotalNPCs, 1);
+
+	AddExpectedActorCount(SimulatedPlayerBPClass, ExpectedPlayers, 1);
+
+	const int32 TotalDropCubes = TotalNPCs + ExpectedPlayers;
+	const int32 DropCubeCountVariance = TotalDropCubes * 0.15f;
```

Comment:
15% just a finger in air number here?

Reply:
Yeah, we could experiment with going a bit tighter. For now just whatever we deem reasonable. Any ideas?

Reply:
Work checking what the variance is in one of the upper level stress tests - this can be done by just looking at the num of entities in a deployment, as that's the only actor that's continually created/destroyed.

Reply:
Looking at graphana metrics I saw a roughly 4.5% variance in two deployments. I have set the allowed variance to 8% to be safe.